meteor/server/api/playout/take.ts

```diff
@@ -56,15 +57,22 @@ export async function takeNextPartInnerSync(cache: CacheForPlayout, now: number)
 	const pBlueprint = pShowStyle.then(async (s) => loadShowStyleBlueprint(s))
 
 	if (currentPartInstance) {
-		const allowTransition = previousPartInstance && !previousPartInstance.part.disableOutTransition
+		const allowTransition = previousPartInstance && !previousPartInstance.part.disableNextInTransition
 		const start = currentPartInstance.timings?.startedPlayback
 
+		if (currentPartInstance.blockTakeUntil && currentPartInstance.blockTakeUntil > getCurrentTime()) {
+			// Adlib-actions can arbitrarily block takes from being done
+			const remainingMs = currentPartInstance.blockTakeUntil - getCurrentTime()
```

Comment:
Minor note: A better variable name would be `remainingTime`.

Also, store `getCurrentTime()` in a variable first, to ensure that it'll be the same value in the if-statement and here.

Reply:
Ive made both changes
I think I didn't bother with storing it in a variable as the second use is only for a debug log so if its off by a couple ms it doesnt matter.